docs/sources/release-notes/v2.5.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Grafana Mimir version 2.5 release notes"
+menuTitle: "V2.5 release notes"
+description: "Release notes for Grafana Mimir version 2.5"
+weight: 300
+---
+
+# Grafana Mimir version 2.5 release notes
+
+Grafana Labs is excited to announce version 2.5 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
```

Comment:
This had a big impact to us, like -50% Alertmanager memory utilization reduction if you have hundred plus tenants:
``` 
* [ENHANCEMENT] Alertmanager: reduced memory utilization in Mimir clusters with a large number of tenants. #3309
```

Reply:
Thanks, I added it to the section about lower memory usage: 2400c6952

Or do you think that this should get a separate item?

Reply:
That's fine! Thanks 🤗